.github/workflows/build-and-test.yml

```diff
@@ -123,6 +123,13 @@ jobs:
           ls -la dist
           pip install $(find dist -name "natcap.invest*.whl")
 
+      - name: debug
+        run: |
+          ls data/invest-test-data
+          ls data/invest-test-data/wind_energy/smoke
+          ls /Users/runner/work/invest/invest/tests/../data/invest-test-data/wind_energy/smoke
+          gdalinfo /Users/runner/work/invest/invest/tests/../data/invest-test-data/wind_energy/smoke/landpoly_smoke.shp
+

```

Comment:
Hey @emlys, should we remove this debug portion now?

Reply:
Oops! Thanks for catching that